### PR TITLE
Brazilian Portuguese translation!

### PR DESCRIPTION
**Pull request by bsides**
Pretty much it. Please let me know if anything else is needed! 😸

### CODE REVIEW

**Comment by sdras** [comment]
Hi! Thanks for stopping by. Sorry for my ignorance but is there a big difference between the Portugese we already have and Brazilian portugese? If not, I'll close this out, but if so, please also translate the /locale/ index. Thanks!

File: store/ptbr/index.js [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+        name: 'splice',
+        shortDesc: 'elemento/s em uma array',
+        desc: 'Adiciona e/ou remove elementos de uma array.',
+        example: `arr.splice(2, 1);<br>
+        console.log(arr);`,
+        output: `[5, 1]`
+      },
+      {
+        name: 'pop',
+        shortDesc: 'O último elemento da array',
```
Comment (TiagoDanin) [inline store/ptbr/index.js]: Start with lowercase letter

File: store/ptbr/index.js [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+        shortDesc: 'retorna uma string representativa da array.',
+        desc: 'Retorna uma string representando a array e seus elementos.',
+        example: `console.log(arr.toString());`,
+        output: `"5,1,8"`
+      },
+      {
+        name: 'toLocaleString',
+        shortDesc: 'retorna uma string localizada representando a array.',
+        desc:
+          'Esse é um pouco esquisito. Retorna uma string localizada representando a array e seus elementos. É super útil para datas e moeda e tem algumas abstrações nativas, então o melhor é consutlar o manual quando for usá-la',
```
Comment (TiagoDanin) [inline store/ptbr/index.js]: Replace `consutlar` >> `consultar`

**Comment by TiagoDanin** [comment]
Yes, especially the vocabulary

**Comment by bsides** [comment]
Sorry guys, when I forked it didn't have a translation yet! This went down pretty quick, uh? Also, I do believe the translation that is going on right now is brazilian, not just portuguese. Yes, they are a bit different, but nothing to worry too much about it.

You can close this!

**Comment by sdras** [comment]
Ok, thank you for the effort @bsides! I'm coming out with an Object Explorer this Tuesday if you want to help contribute over there instead :)

**Comment by bsides** [comment]
Sure thing @sdras thank you 😸

**Pull request closed without merging**